fetch_and_process_api_call/fn-call.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "code_file": "tmp/x.wasm",
+        "function": "myOracleFunc",
+        "input": {
+            "market": "Binance",
+            "coin_id": "bitcoin"
+        },
+        "secret": {
+            "api_key": "CG-NqKT8uUEv9tfwG62w9EUqF7j"
```

Comment:
Note that is my coin gecko key.  I was planning on rotating it, but, we should figure out how we want to handle this since I don't think we want to give this out to people.

Reply:
OK, I set it up as a place holder